src/lib/arch/aarch64/Instruction_decode.cc

```diff
@@ -548,7 +549,7 @@ void Instruction::decode() {
       } else if (metadata_.operands[0].is_vreg) {
         setInstructionType(InsnType::isVectorData);
       } else if ((metadata_.operands[0].reg >= AARCH64_REG_ZAB0 &&
-                  metadata_.operands[0].reg <= AARCH64_REG_ZT0) ||
+                  metadata_.operands[0].reg < AARCH64_REG_ZT0) ||
```

Comment:
Can `ZT0` be used in a SVE context?

Reply:
`ZT0` is enabled / disabled in the same way as `Z0` but has a fixed width of 512-bits, with the logic for detecting whether a `ZT0` related instruction can / can't be executed done in `instruction_execute` as with all other SME instructions.

Regarding where in a core/implementation `ZT0` based instructions are executed, there is no fixed rule in the spec as far as I can tell.... Given its fixed width, to me it seems more SVE-like than SME hence the grouping seen here. And given we don't have co-processor SME support, theres no offload / seperate chip logic to come into play yet